src/app/services/three.service.ts

```diff
@@ -251,25 +250,6 @@ export class ThreeService {
     this.importManager.parseGLTFGeometry(geometry, callback);
   }
 
-  /**
-   * Load geometries from JSON.
-   * @param name Name of the geometries or group of geometries.
-   * @param json JSON containing all the geometries.
-   */
-  public loadJSONGeometry(name: string, json: string | object) {
```

Comment:
Hi @9inpachi can you remind me why you removed this? I think you said that it seems not to be so well supported, but it is in current versions of threejs I think?

Reply:
It more or less doesn't work. I tried to import some JSON but it threw errors.
Would you like me to give it a try again? But this time with JSON exported from Phoenix itself.

Reply:
So I tested it out and exporting and importing using the newer version works.
I am re-adding this method through the import manager. Should I add methods for export as well?